apiserver/authhttp_test.go

```diff
@@ -56,17 +55,6 @@ type authHTTPSuite struct {
 	extraHeaders map[string]string
 }
 
-func (s *authHTTPSuite) SetUpSuite(c *gc.C) {
```

Comment:
Didn't you just add this in a PR yesterday? Or are you just moving it up the hierarchy?